### PR TITLE
feat(sdk): improve license, mock server and post-install for embedding cli

### PR DESCRIPTION
**Pull request by heypoom**
### Description

Improves the license and post-install flow per [the feedback](https://www.notion.so/metabase/CLI-PR-review-8a9be6a85d214910af165e12b1cdfaf0?pvs=4). It allows skipping the license step, allows assigning multiple tenancy columns to user attributes, generates and auto-installs the mock server for you with your package manager of choice, and adds a step-by-step post-installation guide.

### How to verify

- Run `yarn build-embedding-sdk-cli:watch` in one terminal to build the CLI
- Create an empty React project with Vite
- Run `docker rm -f metabase-enterprise-embedding && yarn add file:../metabase/resources/embedding-sdk && npx @metabase/embedding-sdk-react start`
  - This kills existing Docker container, links the built SDK to your React project, then starts the CLI.
  - This only works if your metabase repo is adjacent to the React project. If not, adjust the `file:..` part

### Demo

[Private Loom](https://www.loom.com/share/2639749306e5416fb20c77925a487e7a?sid=c4d22ef9-0544-48c1-b7f4-abb0029f1d80)

### CODE REVIEW

**Comment by heypoom** [comment]
TODO: [address new feedback](https://www.notion.so/metabase/CLI-PR-review-2d2db46826ec42d8a4165972e9e22c2f?pvs=4)

**Comment by WiNloSt** [comment]
@heypoom Is this expected after skipping the license now? 
image

It seems we need to extract part of [Feature flag and settings for SDK](https://www.notion.so/metabase/Feature-flag-and-settings-for-SDK-8ed424c776194528b6332c5fc2510a0c), that always allow `localhost:*` to master otherwise, we won't be able to test this as [`embedding-app-origin` requires a valid token](https://www.metabase.com/docs/latest/configuring-metabase/environment-variables#mb_embedding_app_origin), so we couldn't test this without a valid token.

**Comment by heypoom** [comment]
@WiNloSt It is supposed to fallback to API keys instead of JWT authentication if the license has been skipped. You're right, I think the problem is that even when we are using API keys for authentication, valid CORS headers are still needed for `localhost:*`.

```ts
  // Fallback to API keys when user switching is not enabled.
  if (userSwitcherEnabled) {
    apiKeyOrJwtConfig += `jwtProviderUri: \`\${BASE_SSO_API}/sso/metabase\`,`;
    imports = `import { AnalyticsContext, BASE_SSO_API } from './analytics-provider'`;
  } else {
    apiKeyOrJwtConfig += `apiKey: '${apiKey}'`;
    imports = `import { AnalyticsContext } from './analytics-provider'`;
  }
```

File: enterprise/frontend/src/embedding-sdk/cli/constants/messages.ts [hunk 1]
```diff
@@ -65,19 +51,26 @@ export const getMetabaseInstanceSetupCompleteMessage = (instanceUrl: string) =>
...
+
+export const SETUP_PRO_LICENSE_MESSAGE = `
+  This tool can set up permissions for multi-tenancy and a mock back-end server that
+  signs users into Metabase to emulate the experience from different tenants.
+
+  To proceed, you will need a Pro license.
+  If you skip, we will proceed without multi-tenancy or SSO.
+`;
+
+export const SDK_LEARN_MORE_MESSAGE = `All done! 🚀 Learn more about the SDK here: ${green(
```
Comment (WiNloSt) [inline enterprise/frontend/src/embedding-sdk/cli/constants/messages.ts:74]: This last step message is unaligned 
image

**Review by WiNloSt — APPROVED** [review]
Overall looks great.

File: enterprise/frontend/src/embedding-sdk/cli/constants/messages.ts (hunk 1 above)
Comment (heypoom) [reply enterprise/frontend/src/embedding-sdk/cli/constants/messages.ts:74]: Good eye! Let me align it

**Pull request merged**